src/metabase/models/serialization.clj

```diff
@@ -314,39 +312,14 @@
 
   `:copy` and `:skip` are vectors of field names. `:skip` is only used in tests to check that all fields were
   mentioned.
-
-  `:transform` is a map from field name to a `{:ser (fn [v] ...) :des (fn [v] ...)}` map with functions to
+  j
   serialize/deserialize data.
 
-  For behavior, see `extract-by-spec` and `xform-by-spec`."
+  For behavior, see `extract-one` and `xform-by-spec`."
```

Comment:
Lack of symmetry here sucks. Hmm perhaps I prefer keeping `by-spec`.

Reply:
or maybe `xform-one`? :)